pypeit/core/wavecal/wvutils.py

```diff
@@ -592,13 +646,14 @@ def xcorr_shift_stretch(inspec1, inspec2, cc_thresh=-1.0, smooth=1.0, percent_ce
     y2 = smooth_ceil_cont(inspec2,smooth,percent_ceil=percent_ceil,use_raw_arc=use_raw_arc, sigdetect = sigdetect, fwhm = fwhm)
 
     # Do the cross-correlation first and determine the initial shift
-    shift_cc, corr_cc = xcorr_shift(y1, y2, smooth = None, percent_ceil = None, use_raw_arc = True, sigdetect = sigdetect, fwhm=fwhm, debug = debug)
-    if corr_cc < cc_thresh:
+    shift_cc, corr_cc = xcorr_shift(y1, y2, smooth = None, percent_ceil = None, do_smooth_ceil=False, sigdetect = sigdetect, fwhm=fwhm, debug = debug)
+    # TODO JFH Is this a good idea? Stretch fitting seems to recover better values
+    if corr_cc < -np.inf: # < cc_thresh:
         return -1, shift_cc, 1.0, corr_cc, shift_cc, corr_cc
     else:
```

Comment:
Don't need the else.  The `return` breaks the control flow.

Reply:
Changed.